README.md

```diff
@@ -1 +1,217 @@
-This guy keeps taking my bagels
\ No newline at end of file
+<!-- PROJECT LOGO -->
+<div align="center">
+<img src="client/assets/mini-bag.png" alt="Bagel Logo" width="250" height="250">
+</div>
+
+<h1 align="center"> Quick Bagel </h1>
+<p align="center">
+"This guy <b>keeps</b> taking my bagels." <i> - Tom from Myspace </i>
+<br/>
+A mobile application for you to <b>streamline</b> errands and pay good deeds forward!
+</p>
+
+<!-- PROJECT SHIELDS -->
+![license](https://img.shields.io/npm/l/nodemon)
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary> Table of Contents </summary>
```

Comment:
Used Github Markup to create a Table of Contents drop down menu.

Reference: https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax